docs/src/main/asciidoc/kafka-streams.adoc

```diff
@@ -84,16 +84,33 @@ Create a new project with the following command:
 
 [source, subs=attributes+]
 ----
-mkdir producer && cd producer && \
 mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
     -DprojectGroupId=org.acme \
     -DprojectArtifactId=kafka-streams-quickstart-producer \
     -Dextensions="kafka" \
-    && cd ..
+    && mv kafka-streams-quickstart-producer producer
```

Comment:
Clever :)

Reply:
So that one was not about being clever, it didn't with the adjustments made to the create command to always create a directory for the project.